src/um/versrc.rs

```diff
@@ -0,0 +1,105 @@
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms
+//! Version Resource definitions
+use shared::minwindef::DWORD;
+use um::winnt::LPWSTR;
+use um::winuser::RT_VERSION;
+pub const VS_FILE_INFO: LPWSTR = RT_VERSION;
+pub const VS_VERSION_INFO: DWORD = 1;
+pub const VS_USER_DEFINED: DWORD = 100;
+pub const VS_FFI_SIGNATURE: DWORD = 0xBD04EFFE;
```

Comment:
```suggestion
pub const VS_FFI_SIGNATURE: DWORD = 0xFEEF04BD;
```

You took the _MAC version!